Watchman.DomainModel/Settings/Queries/Handlers/GetInitEventsQueryHandler.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System.Linq;
+using Watchman.Cqrs;
+using Watchman.Integrations.MongoDB;
+
+namespace Watchman.DomainModel.Settings.Queries.Handlers
+{
+    public class GetInitEventsQueryHandler : IQueryHandler<GetInitEventsQuery, GetInitEventsQueryResults>
+    {
+        private readonly ISessionFactory _sessionFactory;
+
+        public GetInitEventsQueryHandler(ISessionFactory sessionFactory)
+        {
+            _sessionFactory = sessionFactory;
+        }
+
+        public GetInitEventsQueryResults Handle(GetInitEventsQuery query)
+        {
+            var session = _sessionFactory.Create();
```

Comment:
a gdzie using?

przy mongo to jest zbędne, ale co jak będziemy chcieli zmienić baze?
nie chcemy musieć myśleć o tym, czy ktoś nie wrzucił usinga